app/src/main/java/one/mixin/android/ui/home/ConversationListFragment.kt

```diff
@@ -499,20 +527,26 @@ class ConversationListFragment : LinkFragment() {
         lifecycleScope.launch {
             val totalUsd = messagesViewModel.findTotalUSDBalance()
 
+            val bulletinView = headerBinding.bulletView
             val shown = bulletinBoard
                 .addBulletin(NewWalletBulletin(bulletinView, requireActivity() as MainActivity, ::onClose))
                 .addBulletin(NotificationBulletin(bulletinView, ::onClose))
                 .addBulletin(EmergencyContactBulletin(bulletinView, totalUsd >= 100, ::onClose))
                 .post()
-            messageAdapter.setShowHeader(shown, binding.messageRv)
+            checkShowHeader(shown)
         }
     }
 
     private fun onClose(type: BulletinView.Type) {
         val shown = if (type.ordinal < BulletinView.Type.values().size - 1) {
             bulletinBoard.post()
         } else false
-        messageAdapter.setShowHeader(shown, binding.messageRv)
+        checkShowHeader(shown)
+    }
+
+    private fun checkShowHeader(showBullet: Boolean) {
+        headerBinding.bulletView.isVisible = showBullet
+        messageAdapter.setShowHeader(isDesktopLogin || showBullet, binding.messageRv)
```

Comment:
There is a display problem that can't _keep at the top_ of the list after the data is loaded for the first time, which may cause it not to display

Reply:
Maybe don't need `notifyDataSetChanged` [here](https://github.com/MixinNetwork/android-app/blob/160e19d893fbf67e3e5d624d876d032965f5e12a/app/src/main/java/one/mixin/android/ui/common/recyclerview/PagedHeaderAdapter.kt#L49), which is called the first time the data is loaded.

Reply:
> There is a display problem that can't _keep at the top_ of the list after the data is loaded for the first time, which may cause it not to display

An intended behavior triggered by https://github.com/MixinNetwork/android-app/pull/2993/files#diff-d5a3b249a853c91e60c29d0b0565acfbd97b6226498f15901902616a97dc7c47R414, may not be necessary.